### DISCUSSION TITLE
Show file library instead of upload file

### DISCUSSION THREAD

**Discussion by joselcvarela** [directus/directus#5031] [Feature Requests]
Hey guys,
Great work with this project.

I am showing this to me team lead and I noticed this
![image](https://user-images.githubusercontent.com/14039341/114568772-561bde00-9c6c-11eb-91ad-1a56f2fd48df.png)

The problem is in the relation with files, always ask to upload a file by default instead of showing the File Library first.

IMO the File Library should appear first and then the user could upload a file if not exists already.

**Comment by rijkvanzanten** [comment]
> The problem is in the relation with files, always ask to upload a file by default instead of showing the File Library first.

I'm not sure I'm following what you're talking about exactly.. Where is this? 

_Also that red dot in the screenshot really threw me off haha. Thought I had a dead pixel!_

**Reply by joselcvarela** [reply]
_Sorry for the red dot, always make that mistake x)_

For example:
![image](https://user-images.githubusercontent.com/14039341/114571665-c166af80-9c6e-11eb-8de5-fa56c97b1cd6.png)

Instead of show "Drag & Drop", I think the File Library should appear here as list or cards with the possibility to drag and drop files.

Like this:
![image](https://user-images.githubusercontent.com/14039341/114572253-3afe9d80-9c6f-11eb-87e8-6a296d4c69ee.png)

**Reply by benhaynes** [reply]
We could add an interface option to the Image Interface that sets the default click action, be that browse Device or browse File Library. Another option would be to use several circular action buttons on the interface, one for each action (similar to the hover menu once an image is selected).

**Reply by benhaynes** [reply]
A quick mockup of what I've described above...

![Screen Shot 2021-04-13 at 4 26 45 PM](https://user-images.githubusercontent.com/522079/114616646-1c9ba080-9c75-11eb-840d-ca325a92c4ef.png)

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
